packages/tiered/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import EventEmitter from 'node:events';
+import Keyv from 'keyv';
+import type {Options, Options_} from './types';
```

Comment:
What is `Options_` vs `Options` ?

Reply:
Options is the parameters that we receive in the constructor. Options_ are the parameters in this.opts